src/backend/api/backend.did

```diff
@@ -10,6 +10,11 @@ type HistoryAction = variant {
   restore;
 };
 
+type SocialLink = record {
+  platform : text;
+  username : text;
```

Comment:
The canister saves the platform and the username. The frontend builds the full URL based on the platform. This means the canister is not doing any validation on URL. I think this is fine since the display of the full URL is a display-only concern and doesn't affect any business logic.